app/migration/mappers/training_programme_type_mapper.rb

```diff
@@ -0,0 +1,19 @@
+module Mappers
+  class TrainingProgrammeTypeMapper
+    # Maps the ECF1 values to the ECF2 values
+    MAPPING = {
+      "core_induction_programme" => "school_led",
+      "design_our_own" => "school_led",
+      "school_funded_fip" => "provider_led",
+      "full_induction_programme" => "provider_led"
+    }.freeze
+
+    def initialize(value)
+      @value = value
+    end
+
+    def mapped_value
```

Comment:
If you changed `TrainingProgrammeTypeMapper` to `TrainingProgrammeType` then this could be changed to `value`.

That means:

`Mappers::TrainingProgrammeTypeMapper.new(extracted_training_programme).mapped_value`

Would become 

`Mappers::TrainingProgrammeType.new(extracted_training_programme).value` 

Which feels cleaner and a bit less repetitive.

Reply:
`mapped_value` makes it clear that you’re getting the converted type rather than the original. With just `value`, it’s less obvious, and I’d feel the need to double check the code to be sure.